weather/app/views/cityForecast.scala.html

```diff
@@ -0,0 +1,17 @@
+@(forecasts: Seq[models.ForecastResponse])(implicit request: RequestHeader)
+
+@import common.Edition
+
+@for((forecast, index) <- forecasts.zipWithIndex) {
+    <li class="forecast__item forecast__item--@index">
+        <p class="weather__desc">
+            <span class="u-h">The temperature at</span>
+            <span class="weather__time">@forecast.hourString</span>
```

Comment:
Did you consider using a `<dl>` for this instead? Also would be nice to see some schema here too and the time wrapped in a `<time>` element.